test/com/kinja/play/plugins/ClosurePluginSpec.scala

```diff
@@ -81,27 +62,7 @@ class ClosurePluginSpec extends Specification with TestApp {
       running(app) {
         Closure.render(
           "closuretest.listInMap",
-          Map("name" -> Some("Test list"), "map" -> Map("items" -> List(1, "2", 3, 4L, 5, 6)))) === "Test list: 1, 2, 3, 4, 5, 6"
-      }
-    }
-  }
-
-  "Render a soy list in map test page" should {
```

Comment:
There are no render methods with those signatures anymore. They had to go because signatures were ambiguous after erasure due to the change in SoyMaps underlying type. (SoyMap -> Map[String, SoyValue] -> Map[String, Any]). There was one singe use in EmbedDisplayApiController in mantle, that was easily changed to use one of the remaining render methods.